### ISSUE TITLE
Attribute "selected" not implemented ?

### ISSUE THREAD

**Issue by esn01** [bwu-dart-graveyard/polymer_ui_elements#119]
following code causes an error: Class 'PolymerUiMenuItem' has no instance getter 'name'.

```
  <polymer-flex-layout isContainer align="center">
    <div>
      <polymer-ui-sidebar-menu selected="0">
        <polymer-ui-menu-item icon="menu" label="Home"></polymer-ui-menu-item>
        <polymer-ui-menu-item icon="menu" label="Explore"></polymer-ui-menu-item>
        <polymer-ui-menu-item icon="menu" label="Favorites"></polymer-ui-menu-item>
      </polymer-ui-sidebar-menu>
    </div>
    <div flex id="sample_container_id">
      <click-counter count="5"></click-counter>
    </div>
    <div>Column Right</div>
  </polymer-flex-layout>
```

**Comment by zoechi** [comment]
Thank you for reporting this issue! I will take a look at it (probably tomorrow).

**Comment by zoechi** [comment]
Is #120 just a duplicate?
I wasn't able to find a difference.

**Comment by esn01** [comment]
No, it is not.

The first issue was the attribute, I think.

Menu was displayed (without error msg.) after I had removed the attribute from code.

But I got the same error after clicking on one of the menu items.

 

Best regards

 

Slawek

 

Gesendet: Mittwoch, 08. Januar 2014 um 14:25 UhrVon: "Günter Zöchbauer" notifications@github.comAn: ErikGrimes/polymer_ui_elements polymer_ui_elements@noreply.github.comCc: esn01 esn01@web.deBetreff: Re: [polymer_ui_elements] Attribute "selected" not implemented ? (#119)

Is #120 just a duplicate?
I wasn't able to find a difference.
